### PR TITLE
Using new UI by default and removing option for new UI

### PR DESCRIPTION
**Pull request by dhlavac**

### CODE REVIEW

File: lib/config.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ class Config {
     this.enableErrLogging = !!process.env.ENABLE_ERR_LOGGING;
 
     // New codebase
-    this.newUIcodebase = process.env.ENV_TYPE == 'stage-new-ui';
+    this.newUIcodebase = process.env.ENV_TYPE != 'production';
```
Comment (kornys) [inline lib/config.ts:83]: I gues it would be driven by URL and not env type for now. if url is qaproauth then inew ui should be used. cc @Frawless

Comment (kornys) [reply lib/config.ts:83]: or leave it as is, seems prod will be switched to new-ui soon

Comment (Frawless) [reply lib/config.ts:83]: I don't have strong opinion as we will remove it soon after prod will be migrated

Comment (dhlavac) [reply lib/config.ts:83]: I would let it as it is and remove whole switching part after pushing new ui to production

**Comment by kornys** [comment]
changes used in https://github.com/redhat-developer/consoledot-e2e/pull/138, this can be closed

**Pull request closed without merging**